tests/CMakeLists.txt

```diff
@@ -0,0 +1,103 @@
+set(CMAKE_COLOR_MAKEFILE ON)
+#set(CMAKE_VERBOSE_MAKEFILE ON)
+
+# Export compile commands to be used with YouCompleteMe
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+# Flags
+set(cxx_compiler_flags
+  -Wextra
+  -Wall
+  -Wfloat-equal
+  -fms-extensions
+  -fdiagnostics-show-option
+  -Wctor-dtor-privacy
+  -Weffc++
+  -fPIC
+  -fno-strict-aliasing
+  -Woverloaded-virtual
+  -Wold-style-cast
+)
+
+foreach(flag ${cxx_compiler_flags})
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${flag}")
+endforeach()
+
+if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+	# using regular Clang or AppleClang
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pedantic-errors")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wgnu-anonymous-struct")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wnested-anon-types")
+endif()
+
+# if DEBUG, also test coverage
+if(CMAKE_BUILD_TYPE MATCHES "Coverage")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -O0 -fprofile-arcs -ftest-coverage")
+endif()
+
+# uncomment for profiling
+#add_definitions(-pg)
+#set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -pg")
+
+include_directories(../include)
+include_directories(../3rdparty/) # this is here since it is clearer to include re2/utils than just utils
+include_directories(../3rdparty/cudd/include)
+include_directories(../3rdparty/re2)
+include_directories(../3rdparty/simlib/include)
+
+add_executable(tests
+		ord-vector.cc
+		number-predicate.cc
+        synchronized-iterator.cc
```

Comment:
Fix indent. I think there is issue with interleaving `\t` and hardspaces (I suggest using hardspaces only).